packages/app/src/components/HelpModal/HelpModal.tsx

```diff
@@ -61,17 +64,11 @@ function KeyCombination(props: { keyCombination: KeyCombination }) {
   )
 }
 
-const { navigator } = globalThis
-
 async function getGPUDeviceInformation() {
-  const adapter = await navigator.gpu.requestAdapter({
+  const { adapter } = await getWebgpuComponents({
     powerPreference: 'high-performance',
```

Comment:
The power preference is hardcoded here, so `adapterOptions` in `<Root>` don't do anything anymore?